mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -205,6 +205,50 @@ func TestAccContainerNodePool_withWorkloadIdentityConfig(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				Config: testAccContainerNodePool_withWorkloadMetadataConfig(cluster, np),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_container_node_pool.with_workload_metadata_config",
+						"node_config.0.workload_metadata_config.0.mode", "GCE_METADATA"),
+				),
+			},
+			{
+				ResourceName:      "google_container_node_pool.with_workload_metadata_config",
+				ImportState:       true,
+				ImportStateVerify: true,
+				// Import always uses the v1 API, so beta features don't get imported.
```

Comment:
Wait, what? GKE uses the beta client, at least as of this change.

Reply:
This was a copy-paste from a similar test, but it seems like this comment and the verify ignore no longer applies. I've removed them from both.